### PR TITLE
Add wiscalert message and update design

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
[MUMPFL-153](https://jira.doit.wisc.edu/jira/browse/MUMPFL-153): "As a student accessing Local/Emergency Contact, I would like to be reminded of the opportunity to register my cell phone for WiscAlerts, so that I can understand the relationship between these registrations and successfully enable the university to contact me in an emergency."

**In this PR**:
- Add message directing people to the WiscAlert portlet to opt-in
- Cosmetic updates
    - Normal spacing in material forms
    - Repeat blocks of info instead of entire cards
    - Icon buttons with tooltips for edit/delete

### Screenshots
(*note: the WiscAlert message in the first screenshot is from a previous commit*)
screen shot 2017-08-29 at 1 23 00 pm
screen shot 2017-08-31 at 9 39 29 am

### CODE REVIEW

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency.html [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
+        <md-input-container class="md-block" flex-gt-sm>
+          <label ng-if="$index === 0">Primary phone</label>
           <input ng-if="$index === 0" type="tel" name="phone{{$index}}" ng-required="$index === 0" ng-pattern=/^([a-zA-Z0-9\s!-.]{1,30})$/ ng-model="contact.phoneNumbers[$index].value">
           <label ng-if="$index !== 0">Other phone</label>
           <input ng-if="$index !== 0" type="tel" name="phone{{$index}}" ng-pattern=/^([a-zA-Z0-9\s!-.]{1,30})$/ ng-model="contact.phoneNumbers[$index].value">
         </md-input-container>
-        <md-input-container class="md-block no-margin-top" flex-gt-sm>
-          <label ng-if="$index === 0">Type*</label>
+        <md-input-container class="md-block no-spacer" flex-gt-sm>
+          <label ng-if="$index === 0">Type</label>
```
Comment (davidmsibley) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency.html]: If these aren't starred any more, no need for the ng-if

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html [hunk 2]
```diff
@@ -1,49 +1,68 @@
...
+        <span class="md-subhead" flex="80">
+          {{ emergencyPhoneNumbers.emergencyPhoneNumbers[0].value}} ({{emergencyPhoneNumbers.emergencyPhoneNumbers[0].type }})
+        </span>
+        <md-button class="md-icon-button" aria-label="edit phone number" ng-click="edit()">
+          <md-icon>mode_edit</md-icon>
+          <md-tooltip md-delay="500">Edit</md-tooltip>
+        </md-button>
+      </div>
+      <!-- Phone number form -->
+      <div ng-hide="!editMode">
```
Comment (davidmsibley) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html]: is this different that `ng-show="editMode"`?

Comment (thevoiceofzeke) [reply my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html]: Nope, functionally the same. I could change them all to only use `ng-show` or `ng-hide`, I just left most of them alone.

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html [hunk 3]
```diff
@@ -1,49 +1,68 @@
...
+          <md-icon>announcement</md-icon>
+        </div>
+        <div class="notice-description" flex-gt-xs="80">
+          <a class="md-subhead" href="https://my.wisc.edu/portal/p/wisc-alerts-text-messaging/render.uP" rel="nofollow noreferrer" target="_blank">Sign up for WiscAlerts</a>
+          <p>
+            WiscAlerts is the name for UW-Madison's emergency notification system. In the event of a campus emergency, students, faculty and staff will receive timely information
+            and updates about a situation and steps you can take to stay safe.
+          </p>
+          <p>
+            Go to the <a href="https://my.wisc.edu/portal/p/wisc-alerts-text-messaging/render.uP">WiscAlerts app</a> to sign up.
```
Comment (apetro) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html]: Turns out students can't sign up their cell phones for wiscalerts messaging in this way, and instead happily the emergency phone number registered through this app automatically registers into wiscalerts. hurray.

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html [hunk 4]
```diff
@@ -1,49 +1,65 @@
...
+        </form>
+      </div>
+      <!-- WiscAlert notice -->
+      <div class="notice-container" layout="row" layout-align="start center">
+        <div class="notice-icon" flex-gt-xs="20" hide-xs>
+          <md-icon>info</md-icon>
+        </div>
+        <div class="notice-description" flex-gt-xs="80">
+          <a class="md-subhead" href="https://my.wisc.edu/portal/p/wisc-alerts-text-messaging/render.uP" rel="nofollow noreferrer" target="_blank">WiscAlerts</a>
+          <p>By saving your emergency phone number, you will be automatically signed up for WiscAlerts, UW-Madison's emergency notification system.</p>
```
Comment (apetro) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html]: This is okay. Would phrasing it in terms of what happens to the phone number be clearer?

"This phone number will be registered to receive WiscAlerts, UW-Madison's emergency notifications."

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html [hunk 5]
```diff
@@ -1,49 +1,65 @@
...
+          <p>
+            For more information, see the <a href="https://kb.wisc.edu/page.php?id=13839">WiscAlerts FAQ</a>.
```
Comment (apetro) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html]: Open external URL in new tab?

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html (hunk 4 above)
Comment (thevoiceofzeke) [reply my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html]: Yeah, I like your version better.

**Pull request merged**